server/models/User.js

```diff
@@ -1,6 +1,6 @@
 const { bookshelf } = require('./../config/database');
 const { comparePassword,hashPassword } = require('./../services/Bcript')
-
+const moment = require('moment');
```

Comment:
Could we use 'current_timestamp' to avoid manual managing of registration date? (Not request to fix, just suggestion)
https://stackoverflow.com/questions/11400147/mysql-current-timestamp-as-default

Reply:
in the future we will change it when we decide how we will set the date